code/modules/mob/living/simple_animal/hostile/giant_spider.dm

```diff
@@ -312,6 +316,8 @@ Nurse caste procs
 
 /datum/ai/giant_spider/nurse/do_process(time_elapsed)
 	. = ..()
+	if(!body.loc)
```

Comment:
Maybe these checks should go into whatever cycle calls do_process? Seems like none of processing should be happening in nullspace